angularSTEP/src/app/view-profiles/view-profiles.component.ts

```diff
@@ -22,7 +22,8 @@ export class ViewProfilesComponent implements OnInit {
   constructor(
     public fAuth: AngularFireAuth,
```

Comment:
So in other files I've noticed that you have `private fAuth` vs `public fAuth` here. I'm wondering if they should all be one or the other?

Reply:
Oops yes you are right! I totally forgot that I noticed that and should change it. Thank you for pointing it out! I believe that Private vs Public shouldn't actually matter all that much in Typescript, but it is better to be consistent.